### DISCUSSION TITLE
Portfolyo (front-end) için ne gibi projeler yaparak ilerlemeliyiz?

### DISCUSSION THREAD

**Discussion by okandavut** [ademilter/homepage#48] [Q&A] [answered]
Merhaba Adem abi ,
Frontend is ilanlarina basvurdugumuzda portfolyo isteniyor. Bu durumda o istenen portfolyo yani istegi karsilayacak portfolyo nasil hazirlanir ?

Tesekkurler.

**Comment by ademilter** [comment, marked as answer]
Merhaba Okan,

#### Kısa cevap;
İşini severek ve içine sinerek yaparsan para ve şöhret sana doğru gelecektir.

#### Uzun Cevap;
Portfolyo için çok süslü ve büyük siteler yapmana gerek yok. 

Codepen'de güzel va basit örnekler yaparak kendini gösterebilir, portfolyonu zenginleştirebilirsin. Dribbble'da gözüne kestirdiğin, koda dökebileceğin süslü ve animasyonlu işleri uygulayıp paylaşabilirsin. Bu taktikle çok fazla ün yaptım freelance iş aldım. Css-Trick, CodePen gibi bloglarda defalarca yaptığım işler örnek verildi, övgü ile bahsedildi. (*)

Front-end camiası ufak (Türkiye için), aşağı yukarı herkes birbirini tanıyor. Önemli olan aradan sıyrılman ve kabul görmüş olman. Bu kabul, yaptığın işler ve kendini pazarlamanla olacaktır. Zira sadece bilmek de yetmiyor. Düzgün şekilde, doğru yerde sunmak çok önemli.

CV için klasik tarzdan uzak dur. Hala referans adı telefonu yazanlar var. Kendinden bahset, sosyal hesaplarını ver (özellikle codepen ve github şıkır şıkır olsun), ne bildiğini anlat, ileriye yönelik neler öğrenmek istediğinden bahset.

İsteği karşılayacak kadar portföy yaparsan fiyatı her zaman karşındaki belirler. İstenilenden fazlasını yapmışsan fiyatı sen belirlersin unutma.

- (*) [Using the <base> tag with Hosted Assets](http://codepen.io/chriscoyier/post/using-the-base-tag-with-hosted-assets)

**Comment by okandavut** [comment]
Yanıt ben ve bazı arkadaşlar için kesinlikle çok yararlı olucaktır.

Her cümlesi için ayrıca teşekkür ediyorum. 👍 

İyi çalışmalar.
